_data/toc.yaml

```diff
@@ -32,24 +32,22 @@ guides:
     title: "Educational resources"
 - sectiontitle: Develop with Docker
   section:
-    - sectiontitle: Develop your apps on Docker
-      section:
-        - path: /develop/
-          title: App development overview
-        - path: /develop/dev-best-practices/
-          title: App development best practices
-        - sectiontitle: Develop images
-          section:
-          - path: /develop/develop-images/dockerfile_best-practices/
-            title: Best practices for writing Dockerfiles
-          - path: /develop/develop-images/baseimages/
-            title: Create a base image
-          - path: /develop/develop-images/build_enhancements/
-            title: Docker build enhancements for 18.09
-          - path: /develop/develop-images/multistage-build/
-            title: Use multi-stage builds
-          - path: /develop/develop-images/image_management/
-            title: Manage images
+  - path: /develop/
+    title: Overview
+  - path: /develop/dev-best-practices/
+    title: Best practices
+  - sectiontitle: Develop images
```

Comment:
Was wondering if we should use this opportunity to rename this section as `Build images` rather than Develop images. WDYT?